st2mistral/functions/stackstorm.py

```diff
@@ -18,16 +18,104 @@
 from oslo_config import cfg
 from oslo_log import log as logging
 
-from st2mistral import config
-config.register_opts()
-
 from mistral import exceptions as exc
+from st2common.jinja.filters import crypto
```

Comment:
Ack - keeping that in mind, will remedy my development script for Mistral to reflect that.